### DISCUSSION TITLE
How to play video in vlc via multicast?

### DISCUSSION THREAD

**Discussion by ayhan9494** [bluenviron/mediamtx#2741] [Questions]
### Question

Hi everybody.
i want use mediamtx on proxied mode. in mediamtx config file added camera rtsp link and i can play on vlc with this link rtsp://localhost:8554/camera without any problem. now i want play video on other system with multicast. how can i do this?
[mediamtx.txt](https://github.com/bluenviron/mediamtx/files/13463333/mediamtx.txt)

**Comment by aler9** [comment]
Hello, instructions on how to read streams with VLC, RTSP and the multicast transport protocol are here:
https://github.com/bluenviron/mediamtx#vlc

**Comment by ayhan9494** [comment]
this command not work for me in oher pc`s
vlc --network-caching=50 rtsp://localhost:8554/mystream?vlcmulticast 

![untitled](https://github.com/bluenviron/mediamtx/assets/112747938/3fcfa263-4164-4cb0-802f-6866d110a753)
@aler9  @dedeibel @philn

**Reply by aler9** [reply]
The server supports reading streams with RTSP-based multicast, that requires an initial handshake with the server and therefore the explicit server IP. After the handshake, packets are sent with multicast, as you can see by using any network dissector.
Reading streams directly with RTP or using an explicit multicast IP is not supported.

**Comment by ayhan9494** [comment]
Thanks for your consideration... 
are you going to support it in near future? or this is a real limitation could not be done?

**Reply by aler9** [reply]
This is not a limitation. Raw RTP doesn't support multiple tracks, while raw multicast doesn't support authentication or multiple streams. RTSP supports both.

[a comment by an automated account is not shown]
